### PR TITLE
Handled dynamic landing

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/Entity/ElectionRound.php [hunk 1]
```diff
@@ -7,8 +7,8 @@
 use Symfony\Component\Validator\Constraints as Assert;
 
 /**
- * @ORM\Entity
  * @ORM\Table(name="election_rounds")
+ * @ORM\Entity
```
Comment (thomasbisignani) [inline src/Entity/ElectionRound.php]: Why move this annotation under the `@ORM\Table` ?

File: templates/procuration/landing.html.twig [hunk 2]
```diff
@@ -14,39 +14,41 @@
...
-                        Chaque vote compte.
-                    </h1>
-                    <h2 class="text--medium b__nudge--top l__hide--on-mobile b__nudge--bottom-small">
-                        Les élections législatives ont lieu les 11 et 18 juin 2017 en France (3 et 17 juin pour les Français de l'étranger du continent Américain et 4 et 18 juin pour les autres Français de l'étranger).
-                    </h2>
-                    <div class="text--body">Si vous ne votez pas en France métropolitaine, <a href="https://www.service-public.fr/particuliers/actualites/A10598" class="link--white">renseignez-vous sur les dates</a>.</div>
+                    {% if next_election %}
+                        {{ next_election.introduction|raw }}
+                    {% else %}
+                        No coming election, TODO.
```
Comment (thomasbisignani) [inline templates/procuration/landing.html.twig]: The `TODO` text is needed ?

File: src/Entity/ElectionRound.php (hunk 1 above)
Comment (HeahDude) [reply src/Entity/ElectionRound.php]: For consistency (i.e https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L30).

File: templates/procuration/landing.html.twig (hunk 2 above)
Comment (HeahDude) [reply templates/procuration/landing.html.twig]: Not more than the English ;), but I'm making small PRs targeting the `Feature/procuration` branch, to ease review and merge quickly. The branch is still WIP and I'm waiting for some content from Marine.

**Pull request merged**